src/main/java/org/reaktivity/nukleus/sse/internal/stream/SseServerFactory.java

```diff
@@ -102,6 +103,8 @@
     private static final String METHOD_PROPERTY = "method";
     private static final String HEADERS_PROPERTY = "headers";
 
+    private static final int MAXIMUM_LAST_EVENT_ID_SIZE = 256;
+
```

Comment:
maximum length for `string8` is `255` since the range of values is `0` to `255` (not `256`)